autopts/utils.py

```diff
@@ -31,12 +31,14 @@
 import hid
 import psutil
 
+from autopts.config import FILE_PATHS
+
 PTS_WORKSPACE_FILE_EXT = ".pqw6"
 
 # Global paths for wid report
 BASE_DIR = Path(__file__).parent.parent.resolve()
 LOG_DIR = BASE_DIR / "logs"
-OUTPUT_CSV_PATH = BASE_DIR / "tmp" / "wid_usage_report.csv"
+OUTPUT_CSV_PATH = Path(FILE_PATHS['WID_USE_CSV_FILE'])
```

Comment:
This import is done before the bot config.py is parsed, so overriding the path will not work.

Reply:
Thanks :)